data/data/powervs/iaas/providers.tf

```diff
@@ -3,8 +3,16 @@
 terraform {
   required_providers {
     ibm = {
-      source = "ibm-cloud/ibm"
-      version = "~> 1.24.0"
+      source  = "ibm-cloud/ibm"
+      version = "1.25.0"
+    }
+    ignition = {
+      source = "community-terraform-providers/ignition"
+      version = "2.1.2"
+    }
+    presign = {
+      source = "ibm/openshift-powervs/presign"
+      version = "0.0.1"
```

Comment:
do we really need this here in the iaas dir?

Reply:
I had to update the IBM provider and haphazardly copied the `providers.tf` file to all modules. Removed this block, thanks for the catch!